### ISSUE TITLE
Issue after upgrade Chamilo from 1.11.12 to 1.11.14

### ISSUE THREAD

**Issue by MiguelFranco117** [chamilo/chamilo-lms#3765]
**Describe**
After upgrade, when try to complete an exercise and save answare to get score I get a blank screen and nothing else can be done. If I change chamilo site from Production to Dev all works fine.

**To Reproduce**
1. Go to 'courses'
2. Click on 'exersice'
3. Complete the exercise and save it to get score
4. See error of blank screen

**Expected behavior**
Score result screen should be visible

**Screenshots**
How it shows in production mode
![chamilo_issue](https://user-images.githubusercontent.com/78114805/106059994-1dcb2180-60b9-11eb-8f43-f1e7986789a8.jpg)

How it shows in DEV mode
![Expected](https://user-images.githubusercontent.com/78114805/106059935-0e4bd880-60b9-11eb-89ca-c5d69e43aa76.jpg)

**Client Desktop**
 - OS: Windows 10
 - Browser chrome, firefox, edge

**Server**
 - OS: Linux Ubuntu 20.04.3
 - Version of Chamilo:  1.11.14
 - Version of PHP: 7.4.3
 - Version of Apache: 2.4.41
 - Version of MySQL: 8.0.22

Hope someone can help me or guide me.

**Comment by lonesomewalker** [comment]
Hi :-)
Are you able to give the output of error logfile?

**Comment by MiguelFranco117** [comment]
> 
> 
> Hi :-)
> Are you able to give the output of error logfile?

Yes, I'm reading log right now, I'll let you know in some minuts if I found issue and if I could resolve issue

**Comment by MiguelFranco117** [comment]
I could fix the issue. It was originated due two things: 

First, found an entry about some missed permission on cache folder under app directory, so I applied again permissions. 

Second, found an entry saying:

"PHP warning: "continue" targeting switch is equivalent to "break". Did you mean to use "conitnue 2;"? in /var/www/html/chamilo/doctine/orm/lib/Doctrine/ORM/UnitOfWork.php on line 2640"

I change value from "continue" to "break", restarted apache2 services and restarted apache2 service.

I tested again exercise, submitted answers and all seems to be working like a charm. and previous entries iare not being logged anymore.
